database/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "development": {
+    "username": "ilyayudovin",
+    "password": "ilyayudovin123",
+    "database": "spring",
+    "host": "localhost",
+    "dialect": "postgres"
+  },
+  "production": {
```

Comment:
Do NOT commit passwords to the repo. You can use environmental variable in this config. Use it along with .env.